apps/api/src/schema/lib/time.ts

```diff
@@ -53,3 +55,13 @@ export const timeSchema = z
   .regex(TIME_REGEX)
   .openapi({ description: "Time string in 12 or 24 hour format", example: "2:00pm" })
   .transform(transformTime);
+
+export const timeRangeSchema = z
+  .string()
+  .regex(TIME_RANGE_REGEX)
+  .transform((s) => s.split("-"))
+  .pipe(timeSchema.array().length(2))
+  .openapi({
+    description: "Time range; endpoints may be in 12 or 24 hour format",
+    example: "8:00am-14:00",
```

Comment:
This is extremely cursed; arguably times and time ranges should be expressed in 24h format. Unfortunately, people here do not use 24h time.

As a stopgap of sorts, I almost want to say that endpoints should be consistent in format, but not sure whether that will be in the best interest of shipping this feature.

Reply:
yeah sure ship it